scripts/draft-blog-post.js

```diff
@@ -22,7 +22,8 @@ const introFile = path.join(changelogUnreleasedDirPath, "blog-post-intro.md");
 if (!fs.existsSync(introFile)) {
   fs.copyFileSync(introTemplateFile, introFile);
 }
-const previousVersion = require("prettier/package.json").version;
+const previousVersion =
+  require("../node_modules/prettier/package.json").version;
 const version = require("../package.json").version.replace(/-.+/, "");
```

Comment:
Can you also use `semver` to parse it and use major/minor/patch to get the version?

Reply:
Yes, but why? I've pushed new commit [2f5c1dd](https://github.com/prettier/prettier/pull/14636/commits/2f5c1dd00993d8c1cde9adec446b1768f0fc2026). Please check it if it follows your imagine.

Reply:
Just to be safer.